### DISCUSSION TITLE
Recover discarded uncommitted files

### DISCUSSION THREAD

**Discussion by rozerx** [community/community#75342] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I accidentally discarded files while I was trying to change branch and they were uncommitted. How can I recover them? I checked the reflog but I couldn't find solution. I also restored the pc to Friday with Windows restore point but files didn't show up.

**Comment by b8zeek** [comment, marked as answer]
Hey Ramazan, this happened to me, as well and I had something like 8h of work un-committed... I tried many things, everything was unsuccessful. I would advise you to stop searching for the solution as I think you're wasting your precious time and get back to re-writing it. It should take much less time than the first time because you went through the process already.

Sorry this happened to you, I know the feeling. 😞 Good luck. 🍺 

🍀

**Reply by farisei77** [reply]
Good

**Reply by rozerx** [reply]
Thank you for responding @bejzik8 I just started to re-write :/

**Reply by b8zeek** [reply]
You're welcome buddy. Good luck.

🍀

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @rozerx! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by b8zeek** [reply]
So kind you are.

🍀

**Comment by Oil3** [comment]
I knnow we deserve everything that happen to us, but this is really annoying.
